property_app/serializers.py

```diff
@@ -1,16 +1,18 @@
 from rest_framework import serializers
-from .models import Property, Operating_Expenses, Purchase_Worksheet
+from .models import Property, Operating_Expenses, Purchase_Worksheet, List_of_Properties, Portfolio
 
 class PurchaseWorksheetSerializer(serializers.ModelSerializer):
 
-    operating_expenses = serializers.SerializerMethodField()
     matching_property = serializers.SerializerMethodField()
 
     class Meta:
         model = Purchase_Worksheet
         fields = ['id', 'matching_property', 'purchase_price', 'financing', 'interest_rate',
-                  'purchase_cost', 'gross_rent', 'arv', 'down_payment', 'loan_term', 'rehab_cost',
-                  'vancancy_rate', 'operating_expenses']
+                  'purchase_cost', 'gross_rent', 'arv', 'down_payment', 'loan_term',
```

Comment:
if these are all the fields it may be beneficial to specify "_all_" instead of naming each one